### PR TITLE
Some style changes in question comment

### PR DESCRIPTION
**Pull request by ElaPl**

### CODE REVIEW

File: izp/polls/static/css/bootstrap.css [hunk 1]
```diff
@@ -3537,3 +3537,6 @@ button.list-group-item-danger.active:focus {
     display: none !important;
   }
 }
+.top10 {
```
Comment (sgorawski) [inline izp/polls/static/css/bootstrap.css]: Just a suggestion - maybe the same effect could be achieved using the styles that are already implemented in Bootstrap 4, therefore eliminating the need for more custom CSS? Check [this](https://getbootstrap.com/docs/4.0/utilities/spacing/) out.

File: izp/polls/templates/polls/question_detail.html [hunk 2]
```diff
@@ -2,49 +2,52 @@
...
+               </label>
+             </div>
+            {% endfor %}
+            {% if is_open %}
+            <div class="form-group">
+               <label for="new_choice">
+                   Własna propozycja:
+               </label>
+               <input id="new_choice" name="new_choice"
+                      type="text" class="form-control" >
```
Comment (sgorawski) [inline izp/polls/templates/polls/question_detail.html]: Minor aesthetic issue: space before `>`.  Same in line 34

File: izp/polls/static/css/bootstrap.css (hunk 1 above)
Comment (ElaPl) [reply izp/polls/static/css/bootstrap.css]: Yes, I agree with You, but in project we don't use Bootstrap 4. As I know, Valentin Baliuk is responsible for bootstrap  framework in our project. He told that bootstrap 4 is in alpha version so there can be a lot of bugs and bootstrap 3 is simplier to customize.
That's why i did it this way.

File: izp/polls/templates/polls/question_detail.html (hunk 2 above)
Comment (ElaPl) [reply izp/polls/templates/polls/question_detail.html]: Changed

File: izp/polls/static/css/bootstrap.css (hunk 1 above)
Comment (sgorawski) [reply izp/polls/static/css/bootstrap.css]: Mhmmm. Tbh I thought it was Bs4. Guess you learn sth new everyday :)

File: izp/polls/templates/polls/question_detail.html [hunk 3]
```diff
@@ -2,49 +2,52 @@
...
-         <p>{{ comment.text|linebreaks }}</p>
+       </div>
+      <div class="row container-fluid top10">
+         {% for comment in comments %}
+         <div class="comment">
+            <div class="date"><strong>{{ comment.date }}</strong></div>
+            <p>{{ comment.text|linebreaks }}</p>
+         </div>
+         {% empty %}
+            <p>No comments</p>
```
Comment (czapiga) [inline izp/polls/templates/polls/question_detail.html]: Change to `Brak komentarzy` or smth like that

**Pull request merged**